aips/aip-81.md

```diff
@@ -82,30 +82,16 @@ Some concrete VUF schemes applicable here:
 - [Pinkas VUF](https://eprint.iacr.org/2024/198.pdf).
 
 Below we refer to the VUF scheme being used in pepper calculation as `vuf`,
-and the evaluation function as `vuf.eval`.
-
-#### Asymmetric encryption
-
-In addition, an asymmetric encryption with variable-length input is required to encrypt the pepper with end user's ESK,
-to ensure only the end user can see the pepper.
-Otherwise, anyone who has access to a leaked JWT can learn the account address by interacting with the pepper service.
-
-Here is an example asymmetric encryption scheme that combines ElGamal asymmetric encryption scheme and AES-256-GCM `(Aes256Gcm.enc, Aes256Gcm.dec)` symmetric encryption with variable-length input (assuming the ephemeral key pair is an Ed25519 key pair, scheme referred to as `ElGamalCurve25519Aes256Gcm` below).
-- Let `x, Y` be the Ed25519 key pair.
-- To ecrypt variable-lenth input `ptxt`:
-  - Pick a point `M` from the Ed25519's underlying prime-order group uniformly at random.
-  - Encrypt `M` using El-Gamal with public key `Y`. Denote by `C0, C1` the ciphertext, where `C0, C1` are 2 points on Curve25519.
-  - Hash `M` to an AES-256-GCM key `aes_key`.
-  - Encrypt `ptxt` using AES-256-GCM with encryption key `aes_key`. Denote by `ctxt` the variable-length AES ciphertext.
-  - The full ciphertext is `C0, C1, ctxt`.
-- To decrypt `C0, C1, ctxt`:
-  - Recover `M` from `C0, C1` using El-Gamal decryption and private key `x`.
-  - Hash `M` to an AES-256-GCM key `aes_key`.
-  - Decrypt `ptxt` from `ctxt` using AES-256-GCM decryption with `aes_key`.
-  - The final output is `ptxt`.
-
-Below we refer to the asymmetric encryption scheme being used as `asymmetric`,
-and its encryption algorithm as `asymmetric.enc`.
+the evaluation function as `vuf.eval`,
+and the vuf output entropy as `vuf_entropy`.
+
+#### Pepper base hasher
+A cryptographic hash function `H` with output bit-length `hash_bit_len` is needed to hash a VUF output into a bit string,
```

Comment:
Can you just say what the actual hash function is?

Reply:
fixed, mentioned in a "Instantiation in reference implementation" section.